BLISS Section 1.1 on Research Question (#1507)
ihmeuw/vivarium_research: docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst

1.1 Research question
---------------------

When you’re designing a research question, it’s important to consider both if race, 
ethnicity, and/or ancestry are relevant, and, if so, what the casual pathways might be. Spending 
time considering the causal pathways will help you consider what exactly you are 
investigating and might identify more relevant proxy measures.

> To work through this example, let’s consider a starting research question: how will 

Reviewer: I like that CVD is our recurring example throughout this section, it provides clarity/consistency. It could also be helpful to offer an example research question where race/ethnicity/ancestry would not be as relevant to a research question?

the rates of cardiovascular disease (CVD) change over the next 5 years in the US?

First, let’s consider if including race, ethnicity, and/or ancestry is relevant to 
the research question. It’s commonly known that CVD rates vary with race, and that 

> race demographics are changing within the US. Understanding how different groups’ 

Reviewer 2: ```suggestion
racial demographics are changing within the US. Understanding how different groups’ 
```

Not confident in this, I just feel like I've more often heard racial.

CVD rates are changing differently might lead us to make a better forecast for 
the future. So, it seems relevant to include race here.  

Second, let’s think about if this will lead to changes in policies. Here again, 
it seems plausible that if we see a sharper increase  in CVD incidence among some 
racial groups, decision-makers might deploy racially aware intervention strategies. 
So here, including something about race, ethnicity, and/or ancestry seems relevant. 

Last, let’s think through the possible causal pathways and decide what metric(s) 

> we want to include. In the below diagrams, the solid lines indicate casual pathways, 

Reviewer 3: ```suggestion
we want to include. In the below diagrams, the solid arrows indicate casual pathways, 
```
Would this be more accurate?

> while the dotted lines indicate correlation. 

Reviewer 2: The dotted arrows are something else -- I know I suggested them 😅 but maybe we should use color or something else to indicate "we don't like this arrow/it doesn't have to be this way"

**Option 1a: Race** 

.. image:: race_a_v1.svg

> We stated above that its commonly known that CVD rates vary with race. This is 

Reviewer: ```suggestion
We stated above that it's commonly known that CVD rates vary with race. This is 
```

> well documented and true. It's easy therefore to think about the casual diagram 

Reviewer: ```suggestion
well documented and true. It's easy therefore to think about the causal diagram 
```

> as a single line between race and disease. But all we know is that there is a 

Reviewer 3: ```suggestion
as a single arrow from race to disease. But all we know is that there is a 
```
Is this what you mean? Or did you want the arrow to go the other way? Or do you mean a single dotted correlation line?

correlation between race and CVD. We want to more closely examine what might be 
causing this correlation. 

> 

Reviewer 3: I think it would be helpful to start with the smaller diagram showing just the correlation between race and disease, as described in the paragraph above, then start a new section here for Option 1a, showing one possible explanation for the correlation, i.e. the diagram you have above, except I think it would then be clearer to omit the dotted line from that diagram because the common cause of "Physical Traits" is our first hypothesis of what's causing the correlation, so the dotted line is redundant with this common cause.

> Given our defintion of race as physical characteristics, for race to cause the 
> disease, it means that the physical characteristics are causing the disease itself. 

Reviewer 2: ```suggestion
Race is related to physical characteristics.
We might naively assume that the physical characteristics are causing the disease itself. 
```

I don't love saying race = physical characteristics, when they have separate bubbles in this diagram. Suggested simplification.

There might be a few cases where this causal pathway is correct, but they are limited 
in scope. For example, lighter skinned individuals tend to get skin cancer at higher 

> rates. In this case, the physical characteristic of race impacts the disease. However, 

Reviewer 2: ```suggestion
rates. In this case, the physical characteristic which leads to racial categorization impacts the disease. However, 
```

Maybe too wordy?

**Option 1b: Race** 

.. image:: race_b_v1.svg

A much more common option is race “leads” to experiences of racism which in turn leads 
to the disease. We put “leads” in quotes as we don’t want to imply that existing with 
certain physical characteristics “causes” racism, but rather that racism is the result 
of years of history defining groups based on physical characteristics. 

This casual pathway is much more common, and seems more applicable to our example with CVD. 
However, considering the exact pathway here is important. We list a few possible pathways for 
CVD based on race. Note that these do overlap with ethnicity in some cases. This is 
because discrimination can impact culture/ethnic practices. These are often 
rooted in racism. 

- Historical red lining leading to African Americans living in food deserts, having less available green space, or living in neighborhoods with fewer social services
- Historical racism leading to lower wealth in some communities, reduced access to health insurance, or limited access to quality medical care
- Microaggressions experienced by non-white individuals leading to elevated blood pressure 

> - Racism in healthcare settings leading to delayed treatment once a person seeks care 

Reviewer 2: I think these are really helpful examples! I like how they span a wide range of time and scale.

**Option 2a: Ethnicity** 

.. image:: ethnicity_a_v1.svg

> The second causal pathway is between ethnicity and disease. Here, there are 

Reviewer 2: ```suggestion
A second correlation often observed is between ethnicity and disease. Here, there are 
```

Maybe?

> some compelling arguments for CVD. For example, the mediterranean diet has 

Reviewer 2: ```suggestion
some compelling arguments that this relationship could be causal for CVD. For example, the Mediterranean diet has 
```

been shown to lower risk of cardiovascular disease. So ethnic groups that 
tend to follow this type of diet might have lowered risk of CVD. 

> However, a big risk of this causal pathway is that ethnicity can be 
> greatly affected by circumstance and history. 

Reviewer 2: ```suggestion
However, this is not the only potential explanation for correlation with ethnicity,
which can be 
greatly affected by circumstance and history. 
```

I think spelling out the "risk" more explicitly helps clarify here.

**Option 2b: Ethnicity** 

.. image:: ethnicity_b_v1.svg

Here is a more complete picture of the same casual diagram which includes history and 
discrimination. One example of how history impacts ethnicity can be seen by comparing 
immigrant populations' CVD rates with their counterparts who did not immigrate. As immigrants tend 
to assimilate to “western” lifestyles, which often involve more processed and 
fatty foods, they tend to have higher rates of CVD. Here, the outside force is changes in environment and cultural norms, 
leading to changes in lifestyle and food choices, which in turn to lead to 
differences in CVD rates. 

Some other examples include: 

- Slavery creating new African American cultural norms separate from African cultural norms 
- Availability of green space or parks leading to increase exercise in some communities 

> - Food deserts leading to different food practices

Reviewer 2: I feel like these might be a bit confusing as first examples, as they all kind of entangle ethnicity with race. Could we maybe start with a more cut-and-dry example like language discrimination?
